<launch>
  
  <include file="$(find pr2_pick_manipulation)/launch/manipulation.launch" />
  <node pkg="pr2_pick_perception" type="static_tf.py" name="static_tf_service_node" respawn="true" />
  <node pkg="pr2_pick_perception" type="pcl_utilities.py" name="pcl_utilities" respawn="true" />

  <include file="$(find pr2_gripper_grasp_planner_cluster)/launch/demo.launch" />
  
</launch>
